.gitignore

```diff
@@ -5,3 +5,12 @@
 /ecs-init-*
 *.swp
 /ecs.conf
+/.rpm-done
+/.srpm-done
+/BUILD
+/RPMS
+/SOURCES
+/SRPMS
+/ecs-init.spec
+/sources.tgz
+ecs-init/version/version.go
```

Comment:
@jahkeup The generated file contains the git sha. Committing it, by definition, renders it invalid.